src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -24,9 +24,6 @@
         "../../submodules/NuGet.Versioning/src/NuGet.Versioning/compiler/**/*.cs",
         "../../submodules/**/AssemblyInfo.cs"
     ],
-    "preprocess": [
-        "../../ext/compiler/preprocess/Internalization.cs"
```

Comment:
For the medium-term, I'm disabling this. It makes debugging a major pain and we don't need it yet. I could just disable it on my machine but eventually it's going to end up getting checked in (someone will do it) so we may as well know that we're doing it :)